target/linux/lantiq/config-4.4

```diff
@@ -178,7 +179,6 @@ CONFIG_RTL8366_SMI=y
 CONFIG_SCHED_HRTICK=y
 # CONFIG_SCHED_INFO is not set
 # CONFIG_SCSI_DMA is not set
-# CONFIG_SENSORS_LTQ_CPUTEMP is not set
```

Comment:
I think removing this will cause build error on danube and other not xrx200 platforms.
When these platforms select CONFIG_HWMON the build system does not know what to do with CONFIG_SENSORS_LTQ_CPUTEMP. on xrx200 this is no problem because it is selected in its kernel config.

Reply:
The problem is that "make kernel_menuconfig" only takes the the kernel configuration without the selected packages. 
When you remove CONFIG_SENSORS_LTQ_CPUTEMP from the config-4.4 it is ok for the xway target as long as the hwmon package is not selected. If that gets selected it misses a configuration option for CONFIG_SENSORS_LTQ_CPUTEMP. Normally these options are added to the config in generic/config-4.4, but this driver is only available for lantiq. 
I would suggest to manually add this to the config-4.4 file like it was done before.

Reply:
Ok i can do that but i am not convinced. I think if some changes the subtarget the changer should also update the target. I think the PC is smarter then the human and should do the right.

- Add the missing CONFIG options
- update the config lexical.

So if some updates the CONFIG like me he did not know that CONFIG_SENSORS_LTQ_CPUTEMP must be added if in the future someone add CONFIG_HWMON in the other targets.